src/test/java/uk/gov/companieshouse/service/AppealServiceTest.java

```diff
@@ -183,14 +186,30 @@ public void testGetAppealByPenaltyReference_returnsAppeal() {
         assertEquals(TestData.Appeal.Reason.Attachment.size, appeal.getReason().getOther().getAttachments().get(0).getSize());
     }
 
+    @Test
+    public void testGetMultipleAppealByPenaltyReference_returnsAppeal() {
```

Comment:
please rename to reflect list of appeals returned, something like: `testGetAppealsByPenaltyReference_returnsListOfAppeals()`